src/ImageSharp/Dithering/ErrorDiffusion/IErrorDiffuser.cs

```diff
@@ -0,0 +1,34 @@
+﻿// <copyright file="IErrorDiffuser.cs" company="James Jackson-South">
+// Copyright (c) James Jackson-South and contributors.
+// Licensed under the Apache License, Version 2.0.
+// </copyright>
+
+namespace ImageSharp.Dithering
+{
+    using System;
+
+    /// <summary>
+    /// Encapsulates properties and methods required to perfom diffused error dithering on an image.
+    /// </summary>
+    public interface IErrorDiffuser
+    {
+        /// <summary>
+        /// Gets the dithering matrix
+        /// </summary>
+        byte[][] Matrix { get; }
```

Comment:
Will do 😄... We can enhance after this PR is merged. 

Shall I make a start on our new Color struct or do you want to do that?

Get some sleep! 🛌